virtcontainers/persist/manager.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2018 Huawei Corporation
```

Comment:
Global comment for the whole PR:
Make sure you don't have any `2018` copyright for the new files introduced by this PR.

Reply:
Good catch. Let me modify it.